React-Router-Cheatsheet.md

```diff
@@ -0,0 +1,101 @@
+# React Router Cheatsheet
+
+Based on [http://ricostacruz.com/cheatsheets/react-router.html](http://ricostacruz.com/cheatsheets/react-router.html) but updated for v2.0.0 of React Router.
+
+### Basic
```

Comment:
All the major relevant heading apart from the first relevant/major heading must be Heading 2 as @camperbot delimits at each Heading 2 tag i.e. `##`